### DISCUSSION TITLE
Regarding Running the project on a different chain!

### DISCUSSION THREAD

**Discussion by Jain-Sameer** [Balaji-Ganesh/CrowdHelp-Blockchain-based-crowdfunding-platform#35] [General]
Hi! I was trying this project out for understanding and exploring. I wanted to use the Sepolia chain, I chained the API Key to the Alchemy API of my node, changed the network url in codebase to my sepolia url. also the private key, but when I try to run the project. It is stuck on the deploy script. I can see my Alchemy node receiving the requests but nothing is happening.

**Comment by Balaji-Ganesh** [comment]
Thanks for your interest in exploring.
Sorry, I didn't tried with Alchemy. But pretty sure that, this can be done with Infura for different chain (like Sepolia as you've mentioned) with very little changes in `hardhat.config.js` file.
